edx-platform/pearson-certiport-theme/lms/templates/header/navbar-authenticated.html

```diff
@@ -14,6 +14,7 @@
   show_explore_courses = settings.FEATURES.get('COURSES_ARE_BROWSABLE')
   self.real_user = getattr(user, 'real_user', user)
   enable_help_link = settings.FEATURES.get('ENABLE_HELP_LINK')
+  show_programs_tab = configuration_helpers.get_value('SHOW_PROGRAMS_TAB', show_program_listing)
```

Comment:
Programs will not be activated, so please remove this implementation. @sergivalero20

Reply:
@Squirrel18 Ok, got it.